src/HTSPDemuxer.cpp

```diff
@@ -247,6 +247,26 @@ PVR_ERROR CHTSPDemuxer::CurrentSignal ( PVR_SIGNAL_STATUS &sig )
   return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR CHTSPDemuxer::CurrentDescrambleInfo ( PVR_DESCRAMBLE_INFO &info )
+{
+  CLockObject lock(m_mutex);
+
+  info = {0};
```

Comment:
In pvr.hts, we  everywhere zero the structs that are passed to the addon from outside, yes. Usually we use memset, but I prefer modern syntax nowadays. btw, memset would overwrite the vtable of the struct if it would have one.